src/BotwinExtensions.cs

```diff
@@ -130,6 +137,10 @@ private static void ApplyGlobalBeforeHook(IApplicationBuilder builder, BotwinOpt
             }
         }
 
+        /// <summary>
+        /// Adds Botwin to the specified Microsoft.AspNetCore.Builder.IApplicationBuilder with the specified Microsoft.AspNetCore.Routing.IRouter .
```

Comment:
`/// Adds Botwin to the specified <see cref="IApplicationBuilder"/> with the specified <see cref="IRouter"/>.`